User/Class1.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace User
+{
+    public class User
```

Comment:
It's odd that this is in a file called "Class1.cs".  Consider renaming this file to match the class name to follow convention

Reply:
Looking at this again, it looks like this class isn't used in the project- it should probably be removed